Add 4.5-rc1 blog post (#1156)
godotengine/godot-website: collections/_article/release-candidate-godot-4-5-rc-1.md

---
title: "Release candidate: Godot 4.5 RC 1"
excerpt: Godot 4.5 stable release is imminent; let the last round(s) of testing begin!
categories: [pre-release]
author: Thaddeus Crews
image: /storage/blog/covers/release-candidate-godot-4-5-rc-1.webp
image_caption_title: Whimside
image_caption_description: A game by Toadzillart
date: 2025-09-04 12:00:00
---

The final stage of development for Godot 4.5 has arrived: the [Release Candidate](https://en.wikipedia.org/wiki/Software_release_life_cycle#Release_candidate). This means that all of our planned features are in place and the most critical regressions have been tackled, so we're confident that it's now ready for production use.

> However, we can never be 100% sure that the release is ready to be published without very extensive testing from the community. So while Godot 4.5 is now ready for testing on existing projects (always make a copy/backup before upgrading, preferably with version control), we're eager to hear how it fares and whether any new major issues have been left unnoticed until now.

Reviewer: ```suggestion
However, we can never be 100% sure that the release is ready to be published without very extensive testing from the community. So while Godot 4.5 is now ready for testing on existing projects (always make a copy/backup before upgrading, preferably with version control), we're eager to hear how it fares and whether any new major issues have gone unnoticed until now.
```
I'd say is the more common phrase

> However, we can never be 100% sure that the release is ready to be published without very extensive testing from the community. So while Godot 4.5 is now ready for testing on existing projects (always make a copy/backup before upgrading, preferably with version control), we're eager to hear how it fares and whether any new major issues have been left unnoticed until now.

Reviewer 2: I would add a short paragraph below mentioning the upcoming fixes

```suggestion
However, we can never be 100% sure that the release is ready to be published without very extensive testing from the community. So while Godot 4.5 is now ready for testing on existing projects (always make a copy/backup before upgrading, preferably with version control), we're eager to hear how it fares and whether any new major issues have been left unnoticed until now.

While we are confident that we are _nearly_ ready to release. There will be at least one more RC release after this one containing bug fixes for exporting C# projects to certain, older, Android devices and for shipping baked shaders on iOS devices. We have fixes in the queue already and anticipate merging them promptly and releasing an RC2 early next week. If no major regressions are reported with RC1/RC2, we anticipate releasing 4.5 stable shortly after. 
```

Reviewer 3: Should we also mention that exporting C# projects to Android require .NET 9 now[^1]? Or, since the change is not in RC1, do we only mention it in the RC2 blog post?

[^1]: https://github.com/godotengine/godot/pull/110263

Reviewer 2: I would mention it in RC2!

> Please, consider [supporting the project financially](#support), if you are able. Godot is maintained by the efforts of volunteers and a small team of paid contributors. Your donations go towards sponsoring their work and ensuring they can dedicate their undivided attention to the needs of the project.

Reviewer 4: ```suggestion
Please consider [supporting the project financially](#support), if you are able. Godot is maintained by the efforts of volunteers and a small team of paid contributors. Your donations go towards sponsoring their work and ensuring they can dedicate their undivided attention to the needs of the project.
```
